src/ServiceManager.php

```diff
@@ -271,40 +291,8 @@ public function getAllowOverride()
     }
 
     /**
-     * Configure the service manager
-     *
-     * Valid top keys are:
-     *
-     * - services: service name => service instance pairs
```

Comment:
these details are obsolete as we are now providing a proper psalm type.
so imho this can be deleted, right?